engine/src/main/java/org/pentaho/di/repository/RepositoryBowl.java

```diff
@@ -78,4 +78,4 @@ public Repository getRepository() {
   public void setRepository(Repository repository) {
```

Comment:
EOF new line for engine/src/main/java/org/pentaho/di/repository/RepositoryBowl.java